### PR TITLE
Add Early Termination Liability (ETL) info to all applicable resources

### PR DESCRIPTION
**Pull request by rjouhann**
## Description

Add Early Termination Liability (ETL) info to all applicable resources:
- new computed `etl` field
- warning showing etl value after delete
- new page under guides

## Checklist

- [x] tested locally
- [x] added/updated examples
- [x] added/updated docs

### CODE REVIEW

**Comment by rjouhann** [comment]
Example warning:

```
╷
│ Warning: Resource ID: PF-PD-NYC-NYC-3526453-PF - Early Termination Liability (ETL) cost: $200.00
│ 
│ 
╵
╷
│ Warning: Resource ID: PF-L3-CON-3526386 - Early Termination Liability (ETL) cost: $56.13
│ 
│ 
╵
╷
│ Warning: Resource ID: PF-L3-CUST-3526378 - Early Termination Liability (ETL) cost: $233.87
│ 
│ 
╵

```

File: docs/guides/etl.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+## Monitoring ETL fees
+
+It's important to monitor ETL fees and factor them into your infrastructure planning. You can use the PacketFabric portal or API to check the current ETL fees for your services. Additionally, you can access the etl field of resources in your Terraform configuration to keep track of ETL fees programmatically.
+
+## Handling ETL fees when deleting resources
+
+When deleting a PacketFabric resource in Terraform, it's crucial to consider the ETL fees associated with the resource. Be sure to review your contract terms and ETL fees before proceeding with the deletion.
+
+!> **Warning:** You will have an hourly trial period for the first 24 hours after provisioning a service. This means that any order can be canceled in that 24-hour window, and you will only be charged for the hours in which the service was active. Any NRC (None-Recurrent Cost) charges are also dropped. The exception to this rule is cross connects and Colt ports, which will always trigger a full ETL fee.
```
Comment (celerymen) [inline docs/guides/etl.md:39]: Also, Colt VCs, VC marketplace and IX.

File: templates/guides/etl.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Monitoring ETL fees
+
+It's important to monitor ETL fees and factor them into your infrastructure planning. You can use the PacketFabric portal or API to check the current ETL fees for your services. Additionally, you can access the etl field of resources in your Terraform configuration to keep track of ETL fees programmatically.
+
+## Handling ETL fees when deleting resources
+
+When deleting a PacketFabric resource in Terraform, it's crucial to consider the ETL fees associated with the resource. Be sure to review your contract terms and ETL fees before proceeding with the deletion.
+
+!> **Warning:** You will have an hourly trial period for the first 24 hours after provisioning a service. This means that any order can be canceled in that 24-hour window, and you will only be charged for the hours in which the service was active. Any NRC (None-Recurrent Cost) charges are also dropped. The exception to this rule is cross connects and Colt ports, which will always trigger a full ETL fee.
```
Comment (celerymen) [inline templates/guides/etl.md:39]: my comment above applies here too about full etl

**Review by celerymen — APPROVED** [review]
Looks good to me from my point of view

File: docs/guides/etl.md (hunk 1 above)
Comment (rjouhann) [reply docs/guides/etl.md:39]: good catch! @caitlinwheeless for visibility. I will add those

**Pull request merged**